Mage/src/main/java/mage/abilities/keyword/TransformAbility.java

```diff
@@ -142,7 +142,7 @@ public static void transformCardSpellDynamic(Spell spell, Card otherSide, Game g
 class TransformEffect extends ContinuousEffectImpl {
 
     TransformEffect() {
-        super(Duration.WhileOnBattlefield, Layer.CopyEffects_1, SubLayer.CopyEffects_1a, Outcome.BecomeCreature);
+        super(Duration.WhileOnBattlefield, Layer.TransformEffects_0, SubLayer.NA, Outcome.BecomeCreature);
```

Comment:
See above, maybe related to copy effects rework from #10801 by @xenohedron but you don't write any reasons and details.